src/components/Club/Home/Calendar/index.js

```diff
@@ -132,12 +132,11 @@ const Calendar = () => {
         colorCode: colorCode,
         title: newTitle,
         startDate: startDate,
```

Comment:
밑에 선택이 안되는데 밑에도 endDate만
```suggestion
        startDate
```